insights/specs/default.py

```diff
@@ -948,6 +948,46 @@ def is_mod_loaded_for_ss(broker):
     sssd_config = simple_file("/etc/sssd/sssd.conf")
     subscription_manager_id = simple_command("/usr/sbin/subscription-manager identity")  # use "/usr/sbin" here, BZ#1690529
     subscription_manager_installed_product_ids = simple_command("/usr/bin/find /etc/pki/product-default/ /etc/pki/product/ -name '*pem' -exec rct cat-cert --no-content '{}' \;")
+
+    @datasource(Ps, HostContext)
+    def proc_smaps(broker):
+        ps = broker[Ps]
+        data = []
+        for p_id in ps:
+            pid_id = p_id["PID"]
+            try:
+                with open("/proc/{0}/smaps".format(pid_id), "r") as fp:
+                    swap = {}
+                    for line in fp.readlines():
+                        if not len(line):
+                            continue
+                        else:
+                            k, v = line.split(":", 1)
+                            if "00000000" in k or k == "VmFlags" or line[0].isdigit():
```

Comment:
The first part of this line looks hex number, it's incorrect to check it with `isdigit()` since the `[a-f].isdigit()` returns `False` but it's a valid hex.

Reply:
If `sum`ed these values up in the parser, some of the information will be lost.

For example, it's impossible for a rule to get the `swap` used in the specific range of the virtual memory section (eg. `7f4656856000-7f4656857000 r--p 00007000 fd:00 268516389                  /usr/lib64/libcrypt-2.17.so`) of a PID.


and the `VmFlags` should not be skiped.